cirq-core/cirq/ops/classically_controlled_operation.py

```diff
@@ -169,12 +169,31 @@ def not_zero(measurement):
     def _with_measurement_key_mapping_(
         self, key_map: Dict[str, str]
     ) -> 'ClassicallyControlledOperation':
-        keys = [protocols.with_measurement_key_mapping(k, key_map) for k in self._control_keys]
-        return self._sub_operation.with_classical_controls(*keys)
+        return ClassicallyControlledOperation(
```

Comment:
[No change needed] What's the rationale behind preferring this over `sub_op.with_classical_controls`? (Not oppoesed to it, just wondering)

Reply:
Probably a merge artifact. I prefer the other way and will revert it.